pom.xml

```diff
@@ -31,14 +31,14 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
-    <groupId>org.opensearch</groupId>
-    <artifactId>opendistro_security</artifactId>
+    <groupId>com.amazon.opendistroforelasticsearch</groupId>
```

Comment:
@saratvemulapalli The <groupId> has been changed back together with the `group` in `build.gradle` file.